src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/Airship.cs

```diff
@@ -29,12 +26,44 @@ private void Init()
             
             //Adding Inbox updated listener
             MessageCenterClass.Shared().Inbox.AddListener(this);
+
+            UAirship.Shared().DeepLinkListener = this;
```

Comment:
Do you need to remove this line, now that you are handling it below?

Reply:
oops